pontoon/base/static/js/fluent_interface.js

```diff
@@ -613,27 +696,37 @@ var Pontoon = (function (my) {
           var source = object.original || object.string;
 
           if (!source) {
-            return response;
+            return fallback;
           }
 
           var ast = fluentParser.parseEntry(source);
+          var tree;
 
+          // Value: use entire ast
           if (ast.value) {
-            response = this.serializePlaceables(ast.value.elements);
+            tree = ast;
           }
-          // Attributes
-          else {
-            var attributes = ast.attributes;
-            if (attributes && attributes.length) {
-              response = this.serializePlaceables(attributes[0].value.elements);
-            }
+
+          // Attributes: use ast of the first attribute
+          else if (ast.attributes.length) {
```

Comment:
I think checking `attributes.length` isn't required here. That's why we removed the least `else` block.